frontend/next.config.ts

```diff
@@ -0,0 +1,77 @@
+import { NextConfig } from "next";
```

Comment:
We don't have that much of a `next.js` config to **really** need to typecheck it, but I believe it's a nice way to keep types that we use via `next/config` coherent.

Reply:
Makes total sense!